src/Cli/OrgCommands.php

```diff
@@ -120,6 +131,28 @@ public function orgAnalyze($org, $options = ['as' => 'default', 'format' => 'tab
         return $data;
     }
 
+    /**
+     * Get support level from README.md contents.
+     */
+    protected static function getSupportLevel($readme_contents)
+    {
+        $support_level = null;
+        $lines = explode("\n", $readme_contents);
+        $badges = SupportLevel::getSupportLevelBadges();
+        foreach ($lines as $line) {
+            foreach ($badges as $key => $badge) {
+                if (strpos($line, $badge) !== false) {
```

Comment:
Maybe we should look for just the first part of the badge, e.g. `[![Early Access](https://img.shields.io/`, to allow for some variation in formatting?

Reply:
Done, thanks